二つ名システムに入るボタンをぶち壊した

```
[21:29:02] [Server thread/ERROR]: Could not pass event InventoryClickEvent to SeichiAssist v1.5.2
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1893) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:33) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:10) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_212]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_212]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:748) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
Caused by: scala.MatchError: dØÌ_j
[ (of class java.lang.String)
	at com.github.unchama.seichiassist.listener.invlistener.OnClickTitleMenu$.onPlayerClickTitleMenuEvent(OnClickTitleMenu.scala:70) ~[?:?]
	at com.github.unchama.seichiassist.listener.PlayerInventoryListener.onPlayerClickTitleMenuEvent(PlayerInventoryListener.scala:384) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor248.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_212]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_212]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot-1.12.2.jar:git-Spigot-79a30d7-acbc348]
	... 15 more
```

--- a/src/main/scala/com/github/unchama/seichiassist/listener/invlistener/OnClickTitleMenu.scala
+++ b/src/main/scala/com/github/unchama/seichiassist/listener/invlistener/OnClickTitleMenu.scala
@@ -264,6 +264,9 @@ object OnClickTitleMenu {
             MenuInventoryData.setHeadingIndex(uuid, menuType, MenuInventoryData.getHeadingIndex(uuid, menuType).get + PER_PAGE)
             player.openInventory(MenuInventoryData.computePartsShopMenu(player))
         }
+
+      // それ以外のインベントリの名前だった場合何もしない！
+      case _ => ()
     }
   }
 }

--- a/src/main/scala/com/github/unchama/seichiassist/listener/invlistener/OnClickTitleMenu.scala
+++ b/src/main/scala/com/github/unchama/seichiassist/listener/invlistener/OnClickTitleMenu.scala
@@ -114,6 +114,8 @@ object OnClickTitleMenu {
               ),
               "実績メニューを開く"
             )
+
+          case _ =>
         }
 
         // refresh if needed
@@ -121,6 +123,8 @@ object OnClickTitleMenu {
           case Material.EMERALD_ORE | Material.EMERALD =>
             pd.recalculateAchievePoint()
             player.openInventory(MenuInventoryData.computeRefreshedCombineMenu(player))
+
+          case _ =>
         }
         
       case MenuType.HEAD.invName =>
@@ -155,6 +159,8 @@ object OnClickTitleMenu {
             val menuType = MenuInventoryData.MenuType.HEAD
             MenuInventoryData.setHeadingIndex(uuid, menuType, MenuInventoryData.getHeadingIndex(uuid, menuType).get + PER_PAGE)
             player.openInventory(MenuInventoryData.computeHeadPartCustomMenu(player))
+
+          case _ =>
         }
 
       case MenuType.MIDDLE.invName =>
@@ -187,6 +193,8 @@ object OnClickTitleMenu {
             val menuType = MenuInventoryData.MenuType.MIDDLE
             MenuInventoryData.setHeadingIndex(uuid, menuType, MenuInventoryData.getHeadingIndex(uuid, menuType).get + PER_PAGE)
             player.openInventory(MenuInventoryData.computeMiddlePartCustomMenu(player))
+
+          case _ =>
         }
 
       case MenuType.TAIL.invName =>
@@ -219,6 +227,8 @@ object OnClickTitleMenu {
             val menuType = MenuInventoryData.MenuType.TAIL
             MenuInventoryData.setHeadingIndex(uuid, menuType, MenuInventoryData.getHeadingIndex(uuid, menuType).get + PER_PAGE)
             player.openInventory(MenuInventoryData.computeTailPartCustomMenu(player))
+
+          case _ =>
         }
 
       case MenuType.SHOP.invName =>
@@ -263,10 +273,12 @@ object OnClickTitleMenu {
             val menuType = MenuInventoryData.MenuType.SHOP
             MenuInventoryData.setHeadingIndex(uuid, menuType, MenuInventoryData.getHeadingIndex(uuid, menuType).get + PER_PAGE)
             player.openInventory(MenuInventoryData.computePartsShopMenu(player))
+
+          case _ =>
         }
 
       // それ以外のインベントリの名前だった場合何もしない！
-      case _ => ()
+      case _ =>
     }
   }
 }

